test/fixtures/package-exports.ts

```diff
@@ -0,0 +1,22 @@
+import test from "ava";
+import { createRequire } from "node:module";
+
+// Using CommonsJS require since JSON module imports are still experimental
+const require = createRequire(import.meta.url);
+
+test.serial("Package exports: export of package.json", (t) => {
+	t.truthy(require("@ui5/linter/package.json").version);
+});
+
+// Check number of defined exports
+test.serial("Package exports: check number of exports", (t) => {
+	const packageJson = require("@ui5/linter/package.json");
+	t.is(Object.keys(packageJson.exports).length, 2);
+});
+
+// Public API contract (exported modules)
+test.serial("Package exports: @ui5/linter", async (t) => {
```

Comment:
This is the only test which requires a TS build. What about moving it away from the unit tests into an `e2e` test (e.g. at `test/e2e/` similar to what we have already in ui5-cli. The test can still be an ava test, but shouldn't be included into the default set that runs via `npm run unit`. This way it would not create overhead for all other tests.